src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/step/InstallPluginsStep.java

```diff
@@ -25,6 +26,9 @@ public void execute(InstanceConfiguration config)
     {
         if (config.getClusterConfiguration().getPlugins().size() > 0)
         {
+            if(VersionUtil.isEqualOrGreater_6_4_0(config.getClusterConfiguration().getVersion())) {
+                FilesystemUtil.setScriptPermission(config, "elasticsearch-cli");
```

Comment:
i was making the same change, but you beat me to it :).

To be safe, we should probably check to see if the file exists in `FilesystemUtil.setScriptPermission`, as they can change up the scripts or the names again in the next version

I had

```
          String scriptPath = String.format("bin/%s", scriptName);
          File scriptFile = new File(config.getBaseDir() + "/" + scriptPath);
          if (!scriptFile.exists())
          {
              return;
          }
```